src/lib/fast_check.py

```diff
@@ -191,20 +192,14 @@ async def _check_services(self, project_id):
             return None
         return service_names
 
-    async def execute(self) -> FastCheckResult:
-        _check_configuration_flags(self.logging_context, METRICS_CONFIGURATION_FLAGS)
+    async def is_project_ready_to_monitor(self, project_id, ready_to_monitor):
+        tasks = [self._check_services(project_id)]
 
-        project_list = await get_all_accessible_projects(self.logging_context, self.gcp_session, self.token)
-
-        ready_to_monitor = []
-        for project_id in project_list:
-            tasks = [self._check_services(project_id)]
-
-            results = await asyncio.gather(*tasks, return_exceptions=True)
-
-            if all(result is not None for result in results):
-                ready_to_monitor.append(project_id)
+        results = await asyncio.gather(*tasks, return_exceptions=True)
```

Comment:
I think it is not obvious what return one should expect from this list of tasks that you get from check_services.
Not sure how to express it well

Bc I am thinking about this "if all" here.
Do we need result!=None for all of those tasks? If 10 is ok and 1 is None then we don't monitor project? Can you explain this, this probably needs a comment of explanation in the code

Reply:
Like - what the hell is check services and why we need all results to be true?